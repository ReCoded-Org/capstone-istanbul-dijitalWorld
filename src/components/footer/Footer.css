.footerContainer {
  background: rgba(248, 177, 149, 0.603);
  color: #6c567b;
  display: flex;
}

.footerRow {
  margin: 3% 0;
  display: flex;
  justify-content: space-between;
}

.footerBlurbText,
.footerContactText {
  font-size: 1rem;
  line-height: 1.625rem;
  letter-spacing: -0.03125rem;
  color: rgba(108, 86, 123, 0.83);
}
.footerTitle {
  font-weight: bold;
  font-size: 1.5rem;
  line-height: 2.8rem;
  color: #6c567b;
}

.footerIcons {
  padding: 0.3rem;
  font-size: 2rem;
}
