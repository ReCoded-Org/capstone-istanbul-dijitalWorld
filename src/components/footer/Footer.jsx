import React from 'react';
import {
  faTwitter,
  faFacebook,
  faGithub,
  faInstagram,
  faPinterest,
  faYoutube,
} from '@fortawesome/free-brands-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { Container, Row, Col } from 'react-bootstrap';
import { useTranslation } from 'react-i18next';
import './Footer.css';

const Footer = () => {
  const { t } = useTranslation();
  return (
    <footer className="footerContainer">
      <Container>
        <Row className="footerRow">
          <Col className="footerCol" md="5">
            <p className="footerTitle">{t('home.footer.getInTouch.title')}</p>
            <p className="footerBlurbText">{t('home.footer.getInTouch.text')}</p>
          </Col>

          <Col className="footerCol" md="3">
            <p className="footerTitle">{t('home.footer.contactUs.title')}</p>
            <p className="footerContactText">
              {t('home.footer.contactUs.emailAddress')}
              <br />
              {t('home.footer.contactUs.phoneNumber')}
            </p>
          </Col>

          <Col className="footerCol" md="3">
            <p className="footerTitle">{t('home.footer.followUs')}</p>
            <FontAwesomeIcon className="footerIcons" icon={faTwitter} />
            <FontAwesomeIcon className="footerIcons" icon={faFacebook} />
            <FontAwesomeIcon className="footerIcons" icon={faGithub} />
            <br />
            <FontAwesomeIcon className="footerIcons" icon={faInstagram} />
            <FontAwesomeIcon className="footerIcons" icon={faPinterest} />
            <FontAwesomeIcon className="footerIcons" icon={faYoutube} />
          </Col>
        </Row>
      </Container>
    </footer>
  );
};

export default Footer;
