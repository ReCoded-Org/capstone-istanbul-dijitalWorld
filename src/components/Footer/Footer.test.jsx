import React from "react";
import ReactDOM from "react-dom";
import { render } from "@testing-library/react";
import Footer from "./Footer";
import "@testing-library/jest-dom";

it("Renders succesfully.", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Footer />, div);
});

it("First column renders successfully.", () => {
  const { getByTestId } = render(<Footer />);
  expect(getByTestId("col1")).toBeInTheDocument();
});

it("Second column renders successfully.", () => {
  const { getByTestId } = render(<Footer />);
  expect(getByTestId("col2")).toBeInTheDocument();
});

it("Third column renders successfully.", () => {
  const { getByTestId } = render(<Footer />);
  expect(getByTestId("col3")).toBeInTheDocument();
});
