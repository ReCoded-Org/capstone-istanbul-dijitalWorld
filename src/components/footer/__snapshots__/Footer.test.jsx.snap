// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot. 1`] = `
<footer
  className="footerContainer"
>
  <div
    className="container"
  >
    <div
      className="footerRow row"
    >
      <div
        className="footerCol col-md-5"
      >
        <p
          className="footerTitle"
        >
          home.footer.getInTouch.title
        </p>
        <p
          className="footerBlurbText"
        >
          home.footer.getInTouch.text
        </p>
      </div>
      <div
        className="footerCol col-md-3"
      >
        <p
          className="footerTitle"
        >
          home.footer.contactUs.title
        </p>
        <p
          className="footerContactText"
        >
          home.footer.contactUs.emailAddress
          <br />
          home.footer.contactUs.phoneNumber
        </p>
      </div>
      <div
        className="footerCol col-md-3"
      >
        <p
          className="footerTitle"
        >
          home.footer.followUs
        </p>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-twitter fa-w-16 footerIcons"
          data-icon="twitter"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-facebook fa-w-16 footerIcons"
          data-icon="facebook"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-github fa-w-16 footerIcons"
          data-icon="github"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 496 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <br />
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-instagram fa-w-14 footerIcons"
          data-icon="instagram"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-pinterest fa-w-16 footerIcons"
          data-icon="pinterest"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 496 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M496 256c0 137-111 248-248 248-25.6 0-50.2-3.9-73.4-11.1 10.1-16.5 25.2-43.5 30.8-65 3-11.6 15.4-59 15.4-59 8.1 15.4 31.7 28.5 56.8 28.5 74.8 0 128.7-68.8 128.7-154.3 0-81.9-66.9-143.2-152.9-143.2-107 0-163.9 71.8-163.9 150.1 0 36.4 19.4 81.7 50.3 96.1 4.7 2.2 7.2 1.2 8.3-3.3.8-3.4 5-20.3 6.9-28.1.6-2.5.3-4.7-1.7-7.1-10.1-12.5-18.3-35.3-18.3-56.6 0-54.7 41.4-107.6 112-107.6 60.9 0 103.6 41.5 103.6 100.9 0 67.1-33.9 113.6-78 113.6-24.3 0-42.6-20.1-36.7-44.8 7-29.5 20.5-61.3 20.5-82.6 0-19-10.2-34.9-31.4-34.9-24.9 0-44.9 25.7-44.9 60.2 0 22 7.4 36.8 7.4 36.8s-24.5 103.8-29 123.2c-5 21.4-3 51.6-.9 71.2C65.4 450.9 0 361.1 0 256 0 119 111 8 248 8s248 111 248 248z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-youtube fa-w-18 footerIcons"
          data-icon="youtube"
          data-prefix="fab"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
</footer>
`;
